modules/graphics/src/mrb_font.cxx

```diff
@@ -144,19 +68,23 @@ font_size(mrb_state *mrb, mrb_value self)
 }
 
 /*
+ * @method Font#calc_bounds
  * Calculates the string's width and height in pixel using the font.
+ *
  * @param [String] str
+ * @param [Float] line_height
  * @return [Array<Integer>[2]]
  */
 static mrb_value
 font_calc_bounds(mrb_state *mrb, mrb_value self)
 {
   char *str;
-  mrb_get_args(mrb, "z", &str);
+  mrb_float line_height = 1.0;
```

Comment:
1.0 should be just barely okay, usual defaults on the web are `1.2-1.5`. I usually go with `1.3/1.35`.